setup.py

```diff
@@ -1,6 +1,10 @@
 from setuptools import setup, find_packages
+import os
 
-VERSION = "0.0.16"
+version = {}
+with open("./dds_cli/version.py") as fp:
+    exec(fp.read(), version)
+VERSION = version["__version__"]
```

Comment:
```suggestion
from dds_cli.version import __version__ as VERSION
```

Is there anything preventing that approach here?

Reply:
Got the opportunity to test it. Disregard this comment.